about_us_old.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
```

Comment:
@Bhairevi Delete this file if we are not using it. If we want it, we can always get it back from Git history.